test/integration/test_liquid_order_book_tracker.py

```diff
@@ -0,0 +1,116 @@
+    safe_gather,
+)
+
+
+class LiquidOrderBookTrackerUnitTest(unittest.TestCase):
+    order_book_tracker: Optional[LiquidOrderBookTracker] = None
+    events: List[OrderBookEvent] = [
+        OrderBookEvent.TradeEvent
+    ]
+
+    trading_pairs: List[str] = [
+        'ETHUSD',
+        'LCXBTC'
+    ]
+
+    @classmethod
+    def setUpClass(cls):
+        cls.ev_loop: asyncio.BaseEventLoop = asyncio.get_event_loop()
+        cls.order_book_tracker: LiquidOrderBookTracker = LiquidOrderBookTracker(
+            OrderBookTrackerDataSourceType.EXCHANGE_API,
+            trading_pairs=cls.trading_pairs)
+
+        cls.order_book_tracker_task: asyncio.Task = safe_ensure_future(cls.order_book_tracker.start())
+        cls.ev_loop.run_until_complete(cls.wait_til_tracker_ready())
+
+    @classmethod
+    async def wait_til_tracker_ready(cls):
+        while True:
+            if len(cls.order_book_tracker.order_books) > 0:
+                print("Initialized real-time order books.")
+                return
+            await asyncio.sleep(1)
+
+    async def run_parallel_async(self, *tasks, timeout=None):
+        future: asyncio.Future = safe_ensure_future(safe_gather(*tasks))
+        timer = 0
+        while not future.done():
+            if timeout and timer > timeout:
+                raise Exception("Time out running parallel async task in tests.")
+            timer += 1
+            # now = time.time()
+            # next_iteration = now // 1.0 + 1
+            await asyncio.sleep(1.0)
+        return future.result()
+
+    def run_parallel(self, *tasks):
+        return self.ev_loop.run_until_complete(self.run_parallel_async(*tasks))
+
+    def setUp(self):
+        self.event_logger = EventLogger()
+        for event_tag in self.events:
+            for trading_pair, order_book in self.order_book_tracker.order_books.items():
+                order_book.add_listener(event_tag, self.event_logger)
+
+    # def test_order_book_trade_event_emission(self):
+    #     """
+    #     Test if order book tracker is able to retrieve order book trade message from exchange and
+    #     emit order book trade events after correctly parsing the trade messages
+    #     """
+    #     self.run_parallel(self.event_logger.wait_for(OrderBookTradeEvent))
+    #     for ob_trade_event in self.event_logger.event_log:
+    #         self.assertTrue(type(ob_trade_event) == OrderBookTradeEvent)
+    #         self.assertTrue(ob_trade_event.trading_pair in self.trading_pairs)
+    #         self.assertTrue(type(ob_trade_event.timestamp) == float)
+    #         self.assertTrue(type(ob_trade_event.amount) == float)
+    #         self.assertTrue(type(ob_trade_event.price) == float)
+    #         self.assertTrue(type(ob_trade_event.type) == TradeType)
+    #         self.assertTrue(math.ceil(math.log10(ob_trade_event.timestamp)) == 10)
+    #         self.assertTrue(ob_trade_event.amount > 0)
+    #         self.assertTrue(ob_trade_event.price > 0)
```

Comment:
Remember to include this test once `Market` class is ready

Reply:
Sure, current PR doesn't have Market, will uncomment the test in part 3